portfolio/src/main/webapp/charts.js

```diff
@@ -5,6 +5,30 @@ google.charts.setOnLoadCallback(drawCharts);
 function drawCharts() {
   drawStaticPieChart();
   drawLineChart();
+  drawBarChart();
+}
+
+/*Creates interactive bar chart saying if I'm cool.*/
+function drawBarChart() {
+  fetch('/cool-data').then(response => response.json())
+  .then((coolVotes) => {
```

Comment:
nit: looks like the .then() should be indented by 4 spaces? 
https://google.github.io/styleguide/jsguide.html#formatting-function-expressions

Reply:
Thanks for the catch!